src/main/mi-lite.mc

```diff
@@ -0,0 +1,114 @@
+-- Miking is licensed under the MIT license.
+-- Copyright (C) David Broman. See file LICENSE.txt
+--
+-- The mi-lite is a lightweight compiler using the minimal amount of code
+-- needed for bootstrapping. It is used in place of mi in the first
+-- bootstrapping stage to speed up compile times.
+
+include "options.mc"
+include "mexpr/boot-parser.mc"
+include "mexpr/symbolize.mc"
+include "mexpr/type-annot.mc"
+include "mexpr/utesttrans.mc"
+include "ocaml/compile.mc"
+include "ocaml/generate.mc"
+include "ocaml/pprint.mc"
+
+lang MCoreLiteCompile =
+  BootParser +
+  MExprSym + MExprTypeAnnot + MExprUtestTrans +
+  OCamlPrettyPrint + OCamlTypeDeclGenerate + OCamlGenerate +
+  OCamlGenerateExternalNaive
+end
+
+-- NOTE(larshum, 2021-03-22): This does not work for Windows file paths.
+let filename = lam path.
+  match strLastIndex '/' path with Some idx then
+    subsequence path (addi idx 1) (length path)
+  else path
+
+let filenameWithoutExtension = lam filename.
+  match strLastIndex '.' filename with Some idx then
+    subsequence filename 0 idx
+  else filename
+
+let collectLibraries = lam extNameMap : ExternalNameMap.
+  let f = lam libs. lam lib. setInsert lib libs in
+  let g =
+    lam libs. lam impl : ExternalImpl. foldl f libs impl.libraries
+  in
+  let h = lam libs. lam. lam impls. foldl g libs impls in
+  let libs =
+    mapFoldWithKey h (setEmpty cmpString) extNameMap
+  in
+  setToSeq libs
+
+let ocamlCompile : Options -> [String] -> String -> String -> Unit =
```

Comment:
This removes changes made to ocamlCompile in #449

Reply:
It should be fixed now.